lib/Room.js

```diff
@@ -816,6 +816,74 @@ class Room extends EventEmitter
 				break;
 			}
 
+			case 'getPeers':
+			{
+				// Ensure the Peer is already joined.
+				if (!peer.data.joined)
+					throw new Error('Peer not joined');
+
+				// Tell the new Peer about already joined Peers.
+				// And also create Consumers for existing Producers.
+
+				const joinedPeers =
```

Comment:
Most of this code is duplicated in join, we might want to DRY it up a bit.

Reply:
Fixed